README.md

```diff
@@ -2,23 +2,60 @@
 
 `electrode-keepalive` provides an HttpAgent capable of maintaining available sockets using IP address instead of hostname lookup. This allows for graceful handling of dns changes/failover when changes occur in routing configuration. This is particularly important when using keep-alive enabled connections.
 
+It resolves part of the problems in this [issue](https://github.com/nodejs/node/issues/6713)
```

Comment:
Might be better to just include this snippet in the line above (...keep-alive enabled connections.)